README.md

```diff
@@ -8,15 +8,13 @@ Install the requirements under `tf/requirements.txt`. And call `./init.sh` to co
 
 ## Data preparation
 
-In order to start a training session you first need to download trainingdata from http://lczero.org/training_data. This data is packed in tar.gz balls each containing 10'000 games or chunks as we call them. Preparing data requires the following steps:
+In order to start a training session you first need to download training data from https://storage.lczero.org/files/training_data/. This data is packed in a tar file each contains several chunks, where each chunk is one hours worth of training data. Preparing data requires the following steps:
```

Comment:
chunks are games - each tar file contains an hours worth of chunks.

Reply:
Thanks for your clarification!